tests/test_window.py

```diff
@@ -310,3 +312,45 @@ def test_empty_window_name(session: Session) -> None:
         "#{==:#{session_name}," + session.name + "}",
     )
     assert "''" in cmd.stdout
+
+
+@pytest.mark.skipif(
+    has_lt_version("3.0"),
+    reason="needs -e flag for split-window which was introduced in 3.0",
+)
+def test_split_window_with_environment(session: Session) -> None:
```

Comment:
> I'm not familiar with this one, but can take a look (probably tomorrow earliest

Fine if I add, or would you like to? Up to you

> Any specific idea of what kind of different environment variables might be beneficial to test

- One variable (as you have now): `{"ENV_VAR": "pane"}`
- Two or more: `{"ENV_VAR": "pane", "ENV_VAR_2": "another variable"}`

Reply:
I've extended the tests for `Session.new_window` and `Window.split_window`.

Reply:
@zappolowski LGTM, you can squash

Then I will make a release for libtmux and update tmuxp master to point to this